langchain/utilities/wikipedia.py

```diff
@@ -104,12 +106,12 @@ def load(self, query: str) -> List[Document]:
         Run Wikipedia search and get the article text plus the meta information.
         See
 
-        Returns: a list of documents with the document.page_content in PDF format
+        Returns: a list of documents.
 
         """
         page_titles = self.wiki_client.search(query[:WIKIPEDIA_MAX_QUERY_LENGTH])
         docs = []
-        for page_title in page_titles[: self.top_k_results]:
+        for page_title in page_titles[: self.doc_content_chars_max]:
```

Comment:
is page_titles a string or a list of things? was this line supposed to be changed?

Reply:
A good catch! Thanks! Yes, it should be `top_k_results`.